src/client.rs

```diff
@@ -220,7 +220,8 @@ impl Default for Client {
                 method: http::Method::GET,
                 version: http::Version::HTTP_11,
                 headers: http::header::HeaderMap::new(),
-                body: Bytes::new(),
+                body: Option::from(Bytes::new()),
```

Comment:
Some(Bytes::new()) would be easier to read, but what does Some empty body mean as long as nothing has been received yet?
```suggestion
                body: None,
```